pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -205,45 +223,79 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) error {
 		return err
 	}
 
+	var podLabelKey string
+	var podLabelValue string
+
 	// Here we allow switch between knative and vanilla deployment
 	if isvcutils.GetDeploymentMode(annotations, deployConfig) == constants.RawDeployment {
+		rawDeployment = true
+		podLabelKey = constants.RawDeploymentAppLabel
 		r, err := raw.NewRawKubeReconciler(p.client, p.scheme, objectMeta, &isvc.Spec.Predictor.ComponentExtensionSpec,
 			&podSpec)
 		if err != nil {
+			modelStatus = setModelStatusError(v1beta1.PredictorReconcileFailed, "Fails to create NewRawKubeReconciler for predictor")
```

Comment:
I think these are internal reconciliation errors which are not suitable to expose to users. We can generate a kube event with `InternalError` type.

Reply:
actually we are already generating the event at the top level, so I prefer removing this code.
```go
if err := reconciler.Reconcile(isvc); err != nil {
	r.Log.Error(err, "Failed to reconcile", "reconciler", reflect.ValueOf(reconciler), "Name", isvc.Name)
	r.Recorder.Eventf(isvc, v1.EventTypeWarning, "InternalError", err.Error())
}
```